rubygem-smart_proxy_openscap/rubygem-smart_proxy_openscap.spec

```diff
@@ -3,10 +3,11 @@
 %global foreman_proxy_bundlerd_dir %{_datadir}/foreman-proxy/bundler.d
 %global foreman_proxy_pluginconf_dir %{_sysconfdir}/foreman-proxy/settings.d
 %global spool_dir %{_var}/spool/foreman-proxy/openscap
+%global content_dir %{_var}/lib/openscap
```

Comment:
shouldn't this be in a directory related to foreman-proxy, or is it meant to refer to another package's location?